packages/devtools_app/test/scenes/memory/offline.dart

```diff
@@ -4,6 +4,7 @@ import 'package:devtools_app/src/screens/memory/memory_controller.dart';
 import 'package:devtools_app/src/screens/memory/memory_heap_tree_view.dart';
```

Comment:
not from this PR, but can we rename this file to be more reflective of what this file contains? at this point it contains a Scene and helpers. Maybe scene.dart, default_scene.dart, common.dart, shared.dart? etc.